src/webportal/src/app/job/job-view/job-view.component.js

```diff
@@ -286,11 +290,43 @@ const loadJobs = (specifiedVc) => {
     ],
     'deferRender': true,
     'autoWidth': false,
+    'dom': '<\'row\'<\'col-sm-6\'B><\'col-sm-6\'f>>' +
+      '<\'row\'<\'col-sm-12\'tr>>' +
+      '<\'row\'<\'col-sm-5\'i><\'col-sm-7\'p>>',
+    'buttons': [
+      {
+        text: 'Select all',
+        action: function(e, dt) {
+          dt.rows({page: 'current'}).select();
+        },
+      },
+      'selectNone',
+      {
+        text: 'Stop',
+        action: function( e, dt, node, config ) {
+          const rows = dt.rows({selected: true});
+          const res = confirm(`Are you sure to stop ${rows.count()} jobs?`);
```

Comment:
In fact, enable multi-selection in paged list is hard to "sync the UI state with user" in a limited interface, especially multi-stop such a dangerous operation. User often do not know which jobs they selected after a long page traversing/ordering/filtering. Although there is a count is displayed, when the count becomes large, it turns less and less meaningful to user, unless they already have a target count number.

Therefore, could you describe the common case that multi-stop is helpful? I propose we could add some limits/notices to this operation:

1. User can only stop jobs in current page;
2. Limit the maximum job count user can stop;
3. List all selected jobs in the confirm dialog;
4. Selected job always displayed in the table.

Reply:
Some users reported that they need a job management page to stop all jobs with a specific tag like interns' jobs, low priority jobs...
We can implement this by
  1. add tag to the job name
  2. search jobs with the tag
  3. stop all jobs in the search result.

Reply:
Now all jobs will be deselected when page/search changed.